hax/hax/motr/__init__.py

```diff
@@ -224,12 +224,18 @@ def send_entrypoint_request_reply(self, message: EntrypointRequest):
             # case, so we wait until bootstrapping is done that is all the
             # motr services are up, we check the confd status and exclude
             # corresponding confd from the entrypoint reply.
-            active_confds = []
-            if self.spiel_ready:
-                for confd in confds:
-                    if not util.is_confd_failed(confd.fid):
-                        active_confds.append(confd)
-                confds = active_confds
+            # XXX It seems that the confds were reported as started and they
```

Comment:
Is `XXX` reads "To be revisited later"?

If so, maybe it makes more sense to create the ticket so that this work will not be forgotten. I'm afraid that this way of notes in the code doesn't work in a large team.

Reply:
Sure, created https://jts.seagate.com/browse/EOS-25726